src/main/kotlin/gropius/graphql/TypeGraphQLType.kt

```diff
@@ -0,0 +1,13 @@
+package gropius.graphql
+
+import com.expediagroup.graphql.generator.annotations.GraphQLType
```

Comment:
Someone sitting left of me tells me to write, that this can be removed

Reply:
the person left realised that it is necessary, ups